falcon/request.py

```diff
@@ -220,10 +220,9 @@ def __init__(self, env, options=None):
 
         # PERF(kgriffs): if...in is faster than using env.get(...)
         if 'QUERY_STRING' in env:
-            query_str = env['QUERY_STRING']
+            self.query_string = env['QUERY_STRING']
```

Comment:
it makes sense, Request class don't use self.query_string elsewhere without make a test.